spec/helpers/library_helper_spec.rb

```diff
@@ -10,6 +10,6 @@
 #     end
 #   end
 # end
-describe LibraryHelper do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+# describe LibraryHelper do
```

Comment:
I was just trying to get specs pass. (Apparently that was not very easy). Would you recommend rebasing that commit?